docs/README.md

```diff
@@ -25,6 +27,9 @@ Each component talks to the other components via IPC in the form of HTTP-over-UD
 ![New component overview](img/new-component-overview-simple.svg)
 [(Click here for detailed version)](img/new-component-overview-detailed.svg)
 
+## Relation to IoT Edge
+
+The [IoT Edge runtime](https://docs.microsoft.com/azure/iot-edge/iot-edge-runtime) is responsible for interacting with a container engine to manage containerized IoT Edge modules. The code can be found in the [IoT Edge repo](https://github.com/azure/iotedge). It exposes APIs for those containerized Edge modules to request certificates, encrypt/decrypt secrets and obtain SAS tokens for talking to Azure IoT Hub. It implements these runtime operations by talking to the IS, CS and KS on behalf of the modules, ie it forwards these API requests to the identity, cert, and key services after enhancing them with the identity of the module making the request.
 
 # Provisioning and Runtime Operation
 
```

Comment:
The link on line 45 (GH won't let me add a review comment there) is also broken (try it on https://azure.github.io/iot-identity-service/ ), so please fix that too.

Reply:
I see the link is broken on the GitHub Pages version of the docs, but on [the repo ](https://github.com/Azure/iot-identity-service/tree/main/docs) it's working.  I'm not very familiar with Github Pages, but it seems like the _index.md files weren't included in the site while the other regular .md files were.

Reply:
Yes, because I think Jekyll wants them to be called `index.md`, not `_index.md`

Reply:
Ok, I'll rename.